dingo/gw/pipe/importance_sampling.py

```diff
@@ -108,6 +109,15 @@ def _load_event(self):
         event_dataset = EventDataset(file_name=self.event_data_file)
         self.result.reset_event(event_dataset)
 
+    @property
+    def calibration_marginalization_kwargs(self):
+        if self.calibration_model == "CubicSpline":
```

Comment:
I guess we should add an else here that just raises an exception.

Reply:
If it's `None` we just want it to do nothing. For any other string I suppose an exception makes sense.

Reply:
Actually there'd be an error via the argument parser, which only accept `None` or "CubicSpline" at this point.